.github/workflows/gradle.yml

```diff
@@ -68,12 +68,6 @@ jobs:
       - name: Generate Template scaffold
         run: ./gradlew :airbyte-integrations:connector-templates:generator:testScaffoldTemplates --scan
 
-      - name: Format
```

Comment:
this is increasing the build time by between 25-50%. doesn't seem worth it. 

while it allows us to identify a somewhat common problem earlier, it is ultimately one that's really easy to fix, while it delays feedback on ones that are hard to fix. this is bad for our attention. i can fix a formatting problem that someone else pushes in my sleep. for other build failures, really the person who pushed the change is best suited to fix it but they way to so long that they don't notice the problem until way later.

Reply:
this caused more build problems. going to try to fix separately.